packages/components/src/internal/models.ts

```diff
@@ -295,17 +296,34 @@ export interface EditorModelProps {
     cellValues: CellValues;
     colCount: number;
     id: string;
-    isPasting: boolean;
+    isPasting: boolean; // TODO: remove this attribute
     focusColIdx: number;
     focusRowIdx: number;
     focusValue: List<ValueDescriptor>;
-    numPastedRows: number;
+    numPastedRows: number; // TODO: remove this attribute
```

Comment:
these TODOs for this PR or a future dev story?

Reply:
Gonna do this in this PR I think. It's junk data that is never used. It was probably needed way back when, but it's unused now.